ibacm/src/libacm.c

```diff
@@ -80,18 +80,16 @@ int ib_acm_connect(char *dest)
 			      res->ai_protocol);
 		if (sock == -1) {
 			ret = errno;
-			goto err1;
-		}
-
-		((struct sockaddr_in *) res->ai_addr)->sin_port =
-			htobe16(server_port);
-		ret = connect(sock, res->ai_addr, res->ai_addrlen);
-		if (ret) {
-			ret = errno;
-			close(sock);
-			sock = -1;
+		} else {
```

Comment:
Again this does not follow the kernel coding style which is to bail on error and not have the "non-error flow" in an else statement.  Jason do you care about that?

Reply:
I'm not Jason, but we care.